### PR TITLE
Comment fixes and function clean-up for CRM/Financial directory.

### PR DESCRIPTION
**Pull request by saurabhbatra96**
1. Removed @return void statements.
2. Removed empty redundant functions.
3. Added "@todo Add comments." to a file with no comment blocks.

### CODE REVIEW

File: CRM/Financial/BAO/ExportFormat/CSV.php [hunk 1]
```diff
@@ -66,9 +68,6 @@ public function export($exportParams) {
       $this->$func();
     }
 
-    // now do general journal entries
-    $this->exportTRANS();
```
Comment (eileenmcnaughton) [inline CRM/Financial/BAO/ExportFormat/CSV.php]: Is this intentional?

File: CRM/Financial/BAO/ExportFormat.php [hunk 2]
```diff
@@ -105,14 +103,6 @@ public function getFileExtension() {
   }
 
   /**
-   * Override this if appropriate.
-   * @return null
-   */
-  public function getTemplateFileName() {
-    return NULL;
```
Comment (eileenmcnaughton) [inline CRM/Financial/BAO/ExportFormat.php]: I'm assuming the parent does the same thing & this isn't intentionally blocking it?

**Comment by eileenmcnaughton** [comment]
This looks good - I made a couple of line notes but in general

1) did you check that the empty functions you removed weren't blocking a behaviour that would have been otherwise inherited from their parent class?

2) I like the idea of adding a todo comment block to functions without them. I would love to have our CI enforce the existence of comment blocks but getting to the point where the majority of PRs wouldn't fail on pre-existing uncommented functions seems a huge task. It might be possible to do something clever to add lots ot them (@colemanw any thoughts?)

File: CRM/Financial/BAO/ExportFormat.php (hunk 2 above)
Comment (saurabhbatra96) [inline CRM/Financial/BAO/ExportFormat.php]: I believe that the function isn't even being used in the entire Financial directory (grep call returns nothing). Also the current class has no parent. Should be pretty safe to delete this.

File: CRM/Financial/BAO/ExportFormat/CSV.php (hunk 1 above)
Comment (saurabhbatra96) [inline CRM/Financial/BAO/ExportFormat/CSV.php]: If you look at the exportTRANS() function defined just below, it's empty. I assume there is no point in calling an empty function?

This probably crept in while copy pasting from its neighbor IIF.php.

**Comment by saurabhbatra96** [comment]
Perhaps automatically adding an empty comment block before functions? I'm not sure if we would be any better off that way though because the keyword function might've been used multiple times within comments resulting in the dis-figuration of already existing comments.

**Comment by eileenmcnaughton** [comment]
OK - your explanations show you've thought about the things I noticed - so I'm happy to merge this. Fantastic!

Re the comments - my IDE (phpstorm) automates this to some extent - although it does require quite a bit of review. Anyway - we can park that for now.

**Pull request merged**